### ISSUE TITLE
Loss function in regression

### ISSUE THREAD

**Issue by ogencoglu** [PriorLabs/TabPFN#98]
Is there any documentation for what loss functions are available for regression? Does it accept custom differentiable loss definitions (e.g. with gradients and hessians)?

**Comment by LennartPurucker** [comment]
This repository does not include code for training TabPFN but only for the inference, and thus, there is also no documentation about the loss functions for training. 

TabPFN v2 regressions work with custom differentiable loss functions. We trained it with cross-entropy because we treat regression as a classification problem.
 
You can ask your question on our Discord (https://t.co/i8shglzc61) for more details and help. Furthermore, the Extensions Repo (https://github.com/PriorLabs/tabpfn-extensions) could contain more information and community contributions in the future.   

Feel free to open the issue again if you have further questions about the inference code.
